packages/frontend/store/system/types.ts

```diff
@@ -1,14 +1,29 @@
 export const SET_MOBILE_VIEW = 'SET_MOBILE_VIEW';
+export const SET_WINDOW_DIMENSIONS = 'SET_WINDOW_DIMENSIONS';
 
 export type MobileView = boolean;
+export type Width = number;
+export type Height = number;
+export type WindowDimensions = {
```

Comment:
Let's get rid of Width and Height individual declarations and inside WindowDimensions do the following:
```
width: number;
height: number;
```